src/components/apply/avatar.tsx

```diff
@@ -30,17 +31,18 @@ function getAvatarImage(avatar: AvatarType) {
 
 const Avatar = ({ avatar }: AvatarProps) => {
   return (
-    <div className="relative flex aspect-square w-2/5 flex-col items-center justify-end rounded-lg bg-primary-200 pr-[18.896px]">
-      {avatar && (
-        <>
-          <div className="absolute -left-3 top-0 z-50 -rotate-[8.646deg] transform items-center justify-center gap-2.5 border border-slate-200 bg-primary-100 px-1 py-2">
-            <p className="font-sans text-xs font-medium leading-4 text-primary-500">
-              {avatar}
-            </p>
-          </div>
-          <Image src={getAvatarImage(avatar)} alt="Avatar" fill={true} />
-        </>
-      )}
+    <div className="relative h-[160px] w-[160px]">
+      <div className="absolute left-3 top-1">
+        <p className="text-sm text-[#222734]">{avatar}</p>
+      </div>
+      <div className="flex flex-col items-center justify-end">
+        <Image
+          src={getAvatarImage(avatar)}
+          alt="Avatar"
+          width={160}
+          height={150}
+        />
+      </div>
```

Comment:
be careful changing this, it's used in the passport as well!

Reply:
also, please use our tailwind css variables instead of hex values (they should match the figma colour variables)

Reply:
kept both changes since styling differs in passport, but don't think passport is responsive currently

Reply:
passport is kind of responsive lol